grpc.go

```
@@ -70,6 +71,15 @@ func NewGRPCServer(ctx context.Context, name string, logger *logrus.Entry) *grpc
 		)),
 	}...)
 
+	if enableTLS {
+		tlsConfig, err := getTLSConfig()
+		if err !=nil{
+			logger.WithError(err).Fatal("Unable to get tls config")
```

Shouldn't the function return `nil` in this case? (fail early, fail fast)

---

Does .Fatal(...) not terminate the program?

---

I missed the `Fatal()` sorry